webapp/channels/src/components/global_header/left_controls/history_buttons/history_buttons.tsx

```diff
@@ -76,11 +71,10 @@ const HistoryButtons = (): JSX.Element => {
 
     return (
         <HistoryButtonsContainer>
-            <OverlayTrigger
-                trigger={['hover', 'focus']}
-                delayShow={Constants.OVERLAY_TIME_DELAY}
+            <WithTooltip
+                id='upload-tooltip'
```

Comment:
just to avoid duplication lets have this id more closely related to back button

Reply:
feel free to use anything some suggestions
`global_header_history_back`

Reply:
great suggestion :)